.github/workflows/python-app.yml

```diff
@@ -36,7 +37,6 @@ jobs:
         flake8 . --count --exit-zero --max-complexity=10 --max-line-length=127 --statistics
     - name: Test with unittest
       run: |
-        export WORKINGDIR=$(pwd)
-        export PYTHONPATH="${PYTHONPATH}:${WORKINGDIR}/src"
+        python -m pip install -e ./
```

Comment:
You can do this in three steps: build, install, test. Put them in three steps make sure they are executed independently.

Since you have installed `build` in your dependency step, you can use the following to build:

```shell
python -m build .
```

You can also use `pip` to build, if do not install `build` in the dependency step.

```shell
pip wheel -w dist .
```

In the install step, you just do

```shell
pip install dist/*
```

Then put the test command in a test step. You can also combine install and test step. It is usually a good practice to keep the build step separate.